tasks/src/zero-shot-image-classification/data.ts

```diff
@@ -59,7 +59,12 @@ const taskData: TaskDataCustom = {
 			id: "openai/clip-vit-large-patch14-336",
 		},
 	],
-	spaces:  [],
+	spaces:  [
+		{
+			description: "An application that leverages zero shot image classification to find best captions to generate an image. ",
+			id:          "pharma/CLIP-Interrogator",
+		},
+	],
```

Comment:
Could also be useful to add a Space that just does zero-shot image classification. If there isn't one, I can create one tomorrow.

Reply:
There are some like https://huggingface.co/spaces/Datatrooper/zero-shot-image-classification and the very exciting https://huggingface.co/spaces/CVPR/unicl-zero-shot-img-recog 

Found them from a search + sort https://huggingface.co/spaces?sort=likes&search=zero